### ISSUE TITLE
JAWS compatibility

### ISSUE THREAD

**Issue by alexwarren** [textadventures/quest#53]
Compatibility with screen-reading software such as JAWS should be tested. At
the very least the Player should work with JAWS - all screen text should be
read out.  
## Work Item Details

**Original CodePlex Issue:**    [Issue 663](http://quest.codeplex.com/workitem/663)
**Status:** Closed
**Reason Closed:**  Unassigned
**Assigned to:**    alexwarren
**Reported on:**    May 20, 2011 at 4:27 PM
**Reported by:**    alexwarren
**Updated on:** Aug 3, 2011 at 3:36 PM
**Updated by:** alexwarren
**Closed on:**  Aug 3, 2011 at 3:36 PM
**Closed by:**  alexwarren
